library/reminders.php

```diff
@@ -244,6 +245,7 @@ function update_reminders($dateTarget = '', $patient_id = '', $start = null, $ba
                 $patient_id_complete .= "," . $patient['pid'];
             }
         }
+        $collectedReminders = array_merge([], $collectedReminders, $tempCollectReminders);
```

Comment:
this should be ok, but rec testing to ensure ok

Reply:
This is the easy one. I get why some of the others are the way they're used so, won't be so easy to refactor. Another day perhaps!:)